app/lang.js

```diff
@@ -0,0 +1,38 @@
+// This module is a function module; it handles GUI strings for the purpose of internationalisation.
+// A GUI string should be printed as follows in Javascript code:
+//     lang("string_name", [varargs...])
+// For example, if the language file contains the string binding:
+//     message_score : "%s scored %s points."
+// Then the following call:
+//     lang("message_score", "John", 32)
+// Results in the string:
+//     "John scored 32 points."
+//
+// To translate a string in HTML, replace it by:
+//     <span class="text_string_name"></span>
+// Then, in the JS file for that page, run:
+//     string.fillinTextClasses();
+
+requirejs(["app/config"], function (config) {
+	var language = config.constant("LANGUAGE");
+	
+	requirejs.config({
+		config: {
+			i18n: {
+				locale: language
+			}
+		}
+	});
+});
+
+define(['jquery', 'i18n!nls', 'printf'], function ($, i18n, printf) {
```

Comment:
I had to lo look up what i18n stands for, you might want to consider naming it internationalization. So you'd get
`define([...., 'i18!nls', ....], function(..., internationalization, ....){`